### ISSUE TITLE
Prevent the default browser actions when touching an element selectable

### ISSUE THREAD

**Issue by alexherbo2** [Simonwep/selection#60]
The default browser actions, such as following links.

**Comment by Simonwep** [comment]
Just call `preventDefault` in the `move` event. Please read the README or try it out before submitting an issue.

**Comment by alexherbo2** [comment]
Before submitting, I have tried:

``` javascript
selection.on('move', ({ oe }) => {
  oe.preventDefault()
})
```

**Comment by Simonwep** [comment]
Could you provide a JSFiddle where this fails? As in the template
> If the current behavior is a bug, please provide the steps to reproduce and create a [JSFiddle](https://jsfiddle.net/Simonwep/coyepkas/ ).

This will speed up the process!

**Comment by alexherbo2** [comment]
https://jsfiddle.net/alexherbo2/zmv50kbf/

**Steps**: Click on a container.
**Outcome**: Follows links.
**Expected**: Does not follow links.

**Comment by Simonwep** [comment]
I mean, you don't want that the user can _click_ on links (which is literally the only way to use them) so why do you even wrap selectables in links? If the user can't click / use them they're basically useless?!

SelectionJS shouldn't modify or intercept and interactions which a selectable. That'd really confusing. 

Correct me If I'm wrong but I don't get what you're trying to achieve.

**Comment by Simonwep** [comment]
SelectionJS will only start at a given threshold. If the threshold is exceeded the selection will take place. Everything prior to that won't be intercepted.

Since you'd need to call `preventDefault` on `click` (and SelectionJS only starts during `mousemove` which comes after `mousedown`) It won't be possible to do it within this library. You could either not use any links or bind events manually to prevent it.

TLDR; It's only possible to cancel an link-click by intercepting the click event. But the click event is already in progress where the threshold isn't already met. Therefore it's not possible to do that.

See [this](https://stackoverflow.com/questions/10746650/prevent-click-event-after-mousedown-mouseup-event), [this](https://stackoverflow.com/questions/18342747/why-cant-a-mouseup-event-prevent-a-click-event) and [this](https://stackoverflow.com/questions/8643739/cancel-click-event-in-the-mouseup-event-handler) SO question.

You could although just block the selection in `beforestart` if the user starts on a link.
